app/frontend/src/components/admin/ServiceClients.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <v-card>
+    <v-toolbar flat color="grey lighten-3">
+      <v-card-title>Service Clients</v-card-title>
+    </v-toolbar>
+    <v-container>
+      <!-- search input -->
+      <div class="kc-search">
+        <v-text-field
+          v-model="search"
+          append-icon="mdi-magnify"
+          label="Search"
+          single-line
+          hide-details
+          class="pb-5"
+        ></v-text-field>
+      </div>
+      <!-- table alert -->
+      <v-alert v-if="showAlert" :type="alertType" tile dense>{{alertMessage}}</v-alert>
+      <!-- table header -->
+      <v-data-table
+        class="kc-table"
+        :headers="headers"
+        :items="serviceClients"
+        :items-per-page="10"
+        :search="search"
+        :loading="loading"
+        loading-text="Loading... Please wait"
+        :expanded.sync="expanded"
+        item-key="acronym"
+        show-expand
+      >
+        <!-- environments -->
+        <template v-slot:item.dev="{item}">
+          <div v-if="item.dev">
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-icon color="green" v-on="on">check</v-icon>
+              </template>
+              <span>updated: {{ formatEnvDate(item.environments.DEV) }}</span>
+            </v-tooltip>
+          </div>
+        </template>
+        <template v-slot:item.test="{item}">
+          <div v-if="item.test">
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-icon color="green" v-on="on">check</v-icon>
+              </template>
+              <span>updated: {{ formatEnvDate(item.environments.TEST) }}</span>
+            </v-tooltip>
+          </div>
+        </template>
+        <template v-slot:item.prod="{item}">
+          <div v-if="item.prod">
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-icon color="green" v-on="on">check</v-icon>
+              </template>
+              <span>Updated: {{ formatEnvDate(item.environments.PROD) }}</span>
+            </v-tooltip>
+          </div>
+        </template>
+
+        <!-- expanded row -->
+        <template v-slot:expanded-item="{ headers, item }">
+          <td :class="[responsiveCell]"></td>
+          <td :colspan="headers.length - 1" :class="[responsiveCell]">
```

Comment:
You'd have to double check, but seeing as you want the responsiveCell behavior on all table cells, you may be able to just pull that class out into its own div as a parent of it all.
```
<div :class="[responsiveCell]">
  <td ...
  ...
</div>
```

Reply:
not sure i understand. the class '.v-data-table__mobile-table-row' needs to be applied to all <td> elements on small screens. thats how the vuetify datatable columns are responsive